File: springboot-backend/src/main/java/backend/dto/response/NotificationResponse.java
package backend.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class NotificationResponse {
    private Long id;
    private String title;
    private String message;
    private boolean isRead;
    private LocalDateTime createdAt;
}

File: Fontend/app/src/main/java/com/example/data/dto/response/NotificationResponse.kt
package com.example.data.dto.response

import com.google.gson.annotations.SerializedName

data class NotificationResponse(
    @SerializedName("id")
    val id: Long,

    @SerializedName("title")
    val title: String,

    @SerializedName("message")
    val message: String,

    @SerializedName("read")
    val isRead: Boolean,

    @SerializedName("createdAt")
    val createdAt: String
)